docs/book/src/developer/providers/migrations/v1.4-to-v1.5.md

```diff
@@ -38,6 +38,8 @@ maintainers of providers and consumers of our Go API.
 - Introduced function `CollectInfrastructureLogs` at the `ClusterLogCollector` interface in `test/framework/cluster_proxy.go` to allow collecting infrastructure related logs during tests.
 - A `GetTypedConfigOwner` function has been added to the `sigs.k8s.io./cluster-api/bootstrap/util` package. It is equivalent to `GetConfigOwner` except that it uses the cached typed client instead of the uncached unstructured client, so `GetTypedConfigOwner` is expected to be more performant.
 - `ClusterToObjectsMapper` in `sigs.k8s.io./cluster-api/util` has been deprecated, please use `ClusterToTypedObjectsMapper` instead.
+- The generated `kubeconfig` by the Control Plane providers, as well the optional Certificate Authority stored as secrets, must be labelled with the key-value pair `cluster.x-k8s.io/cluster-name=${CLUSTER_NAME}`.
```

Comment:
```suggestion
- The generated `kubeconfig` by Control Plane providers, as well as the optional Certificate Authority stored as secrets, must be labelled with the key-value pair `cluster.x-k8s.io/cluster-name=${CLUSTER_NAME}`.
```

@prometherion sorry for the back and forth, realized my first suggestion was incomplete when I hit enter

Reply:
Probably will go away :) > https://github.com/kubernetes-sigs/cluster-api/pull/9080#discussion_r1275294580

Reply:
Correct, thanks, Stefan!